social_network_api/src/main/java/com/danit/socialnetwork/config/SecurityConfig.java

```diff
@@ -0,0 +1,40 @@
+package com.danit.socialnetwork.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.Customizer;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.web.cors.CorsConfiguration;
+import org.springframework.web.cors.CorsConfigurationSource;
+import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Configuration
+@EnableWebSecurity
```

Comment:
We already have this annotation. Please, add these beans to the existing class.

Reply:
@miegami I added @EnableWebSecurity and two beans to the existing JavaMailSenderConfig class, but why we cannot have to configuration classes? It is not clear at all for me.

Reply:
@miegami I added two CORS beans to WebSecurtityConfig